docs/src/formula.md

```diff
@@ -0,0 +1,76 @@
+```@meta
+CurrentModule = StatsModels
+DocTestSetup = quote
+    using StatsModels
+end
+```
+
+# Modeling tabular data
+
+Most statistical models require that data be represented as a Matrix-like
+collection of a single numeric type.  Much of the data we want to model,
+however, is __tabular data__, where data is represented as a collection of
```

Comment:
This could get real ugly 😜, but here's why I think it's appropriate to use "data is" here:

To me (and AFAIK, to many if not most English speakers), "data" is a mass noun, like "water" or "information". The ever-objective wikipedia has a [good summary](https://en.wikipedia.org/wiki/Data_(word)) of where this usage is dominant and where it's not (mostly technical writing, especially in the UK, except computer science and related disciplines). I'd like as much as possible to keep documentation in a colloquial style to make it as approachable as possible, and "data is" is more consistent with that style.

For a little linguistic pedantry, as you know "data" is a plural of Latin "datum". But we don't speak Latin, we speak English (and the Latinate singular form of "datum" is so rare that it hardly registers on Google ngrams). In general, it's really common for grammatical things to get mixed up in borrowings like this. In particular, there are plenty of other words that were plural in Latin that are treated as singular when borrowed in English, like "agenda" (plural of "agendum").